lib/node.js

```diff
@@ -118,6 +118,7 @@ declare class Buffer extends Uint8Array {
 
 declare module "buffer" {
   declare var kMaxLength: number;
+  declare export var Buffer: typeof Buffer;
```

Comment:
what does your test look like? it should be something that generates an (expected) error so you can be sure that it's not just acting like `any`/unbound